components/Card.js

```diff
@@ -0,0 +1,216 @@
+export default class Card extends HTMLElement {
+  constructor(props) {
+    super();
+
+    this.props = props;
+    this.dummyLink = "/static/public/image-dummy.png";
+    this.originalDate = new Date(props.createdAt);
+    this.createDate = this.originalDate.toLocaleDateString().slice(0, -1);
+    this.currentDate = new Date();
+    this.timeDiffMinutes = (this.currentDate - this.originalDate) / 1000 / 60;
+
+    this.prettyTimeDiff = (timeDiffMinutes) => {
+      const r = (t) => Math.round(t);
+
+      if (r(timeDiffMinutes) < 2) {
+        return "1 minute ago";
+      } else if (r(timeDiffMinutes) <= 59) {
+        return `${r(timeDiffMinutes)} minutes ago`;
+      } else if (r(timeDiffMinutes) <= 119) {
+        return "1 hour ago";
+      } else if (r(timeDiffMinutes / 60) <= 23) {
+        return `${r(timeDiffMinutes / 60)} hours ago`;
+      } else if (r(timeDiffMinutes / 60) <= 47) {
+        return "1 day ago";
+      } else if (r(timeDiffMinutes / 60 / 24) <= 30) {
+        return `${r(timeDiffMinutes / 60 / 24)} days ago`;
+      } else if (r(timeDiffMinutes / 60 / 24) <= 61) {
+        return "1 month ago";
+      } else if (r(timeDiffMinutes / 60 / 24 / 31) <= 11) {
+        return `${r(timeDiffMinutes / 60 / 24 / 31)} months ago`;
+      } else {
+        return `${Math.floor(timeDiffMinutes / 60 / 24 / 31 / 12)} years ago`;
+      }
```

Comment:
```suggestion
      swich(true) {
        case r(timeDiffMinutes) < 2:
          return ....
      }
```
(취향의 영역) 이런식으로 좀 더 가독성을 높이는 방법도 있으니 참고만 해두세요.

Reply:
사실 switch로 하려다가 막혀서 바꿨는데 true로 채워넣으면 되는 건지 몰랐었습니다! 한번 적용해보겠습니다. 감사합니다!